tex/alg-geom/sheaves.tex

```diff
@@ -501,6 +520,16 @@ \section{Sheaves}
 		This is because to verify a function is continuous,
 		one only needs to look at small open neighborhoods at once.
 
+		\ii Let $X = \RR$, and define the presheaf of rings $\SF$ by:
+		\[
+			\SF(U) = \{ f \colon U \to \RR: \text{there exists continuous $g
+			\colon \RR \to \RR$ such that $g\restrict{U} = f$} \}.
+		\]
+		Then $\SF$ is not a sheaf. Indeed, $s_1(x) = 0$ in $\SF((-1, 0))$
+		and $s_2(x) = 1$ in $\SF((0, 1))$ agrees on the (empty) overlap,
+		but they cannot be glued together to an element in $\SF((-1, 0) \cup(0,
+		1))$.
```

Comment:
Another misconception I had for a while. Can't help to have more red boxes.

(in a similar vein, the function f(p) = 1 if p is on x-axis, 0 if p is on y-axis **is** regular on D(x+y) of V(xy) ⊆ 𝔸² -- i.e. the quasiaffine scheme that is the distinguished open set of the union of x-axis and y-axis -- even though it doesn't "look" continuous. But I can't really find a good place to fit this example in...?)

I suppose it's another case where "geometric intuition works better than algebraic calculation" -- if you try hard enough you can derive an explicit expression for the function above -- namely, x/(x+y).